setup.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "optimade",
+    "version": "0.2.0",
+    "url": "https://github.com/Materials-Consortia/optimade-python-tools",
+    "license": "MIT",
+    "author": "OPTiMaDe Development Team",
+    "author_email": "dev@optimade.org",
+    "description": "Tools for implementing and consuming OPTiMaDe APIs.",
+    "keywords": "optimade jsonapi materials",
+    "include_package_data": true,
+    "package_data": {"": ["*.lark"]},
+    "classifiers": [
+        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Intended Audience :: Developers",
+        "Topic :: Database",
+        "Topic :: Database :: Database Engines/Servers",
+        "Topic :: Database :: Front-Ends"
+    ],
+    "python_requires": ">=3.7",
+    "install_requires": [
+        "pymongo>=3.8",
+        "lark-parser>=0.7.7",
+        "mongomock>=3.16",
+        "fastapi[all]>=0.42.0"
+    ],
+    "extras_require": {
+        "dev": ["black", "invoke", "pre-commit", "twine"],
```

Comment:
In pre-commit we invoke `black` (see [here](https://github.com/Materials-Consortia/optimade-python-tools/blob/master/.pre-commit-config.yaml)).
I would strongly recommend you to use pre-commit to keep commits being "blackened" while working. Pre-commit does not allow you to commit if `black` fails, and will indeed update the changed files so that you can `git add` them and commit, either via your IDE or a terminal.

Concerning `invoke`, I was in doubt. I found only one use, namely in the `task.py` file as you state, and am unsure whether this was still used or not. I guess we are now using the release method implemented by @ltalirz, so I will remove it. I will check whether we can remove `twine` as well.